app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cp_command, current_file, future_file = command.split()
+    if cp_command != "mv" or len(command.split()) != 3:
```

Comment:
Try this:
a, b, c = "hello world".split()
Because now you:
1. do the unpacking
2. and after validate